### ISSUE TITLE
The app on Google Play is not verifiable

### ISSUE THREAD

**Issue by Giszmo** [coinninjadev/dropbit-android#1]
At the time of writing [this article](https://walletscrutiny.com/posts/2019/11/bropbit/) the app was not verifiable. Could you please share clear build instructions and the code matching the versions found on Google Play.

**Comment by Giszmo** [comment]
3.5 months passed. Any update?

**Comment by nschwermann** [comment]
I think the company folded, saw some news about disruption involving the CEO/founder.

**Comment by Giszmo** [comment]
Yes, https://www.reddit.com/r/DropBit/ is not very trust inducing.

**Comment by nschwermann** [comment]
I briefly worked for the company and on this product. I'm confident that the wallet itself is trust worthy but the FBI has seized the lightning node and has frozen all of the companies assets due to investigation of the CEO for an unrelated Bitcoin project. I doubt there will be further support but I could be wrong.

**Comment by Giszmo** [comment]
I marked Dropbit as defunct for now. If there is any reason why people should restart considering this app again, please open an issue [here](https://gitlab.com/walletscrutiny/walletScrutinyCom/-/issues/new?issue%5Bassignee_id%5D=&issue%5Bmilestone_id%5D=).

**Comment by emanuelb** [comment]
The previous walletscrutiny review shows missing-source case: (version 3.2.0 on google-play while 3.1.1 is the latest version+commit+tag in repo)
https://gitlab.com/walletscrutiny/walletScrutinyCom/blob/a920a50eb4b0f8638e7cedb013a135f9c0a7b0fc/_posts/2019-12-20-com.coinninja.coinkeeper.md

Now the latest version on google-play is newer 3.2.7 last update: January 28, 2020, but the repo not updated & the latest commit is for old 3.1.1 version, opened issue for it at: #2
